nplab/instrument/__init__.py

```diff
@@ -12,34 +12,46 @@
 """
 
 from builtins import str
-from nplab.utils.thread_utils import locked_action_decorator, background_action_decorator
```

Comment:
locked_action and background_action are nicer aliases IMO

Reply:
Could agree, not an issue for this branch :P (it's been like that since the start of nplab)